addons/storyshots/storyshots-puppeteer/package.json

```diff
@@ -54,7 +54,7 @@
   },
   "peerDependencies": {
     "@storybook/addon-storyshots": "6.5.0-alpha.64",
-    "puppeteer": "^2.0.0 || ^3.0.0"
```

Comment:
We should add a devDependency with the version we depended on before, to prevent breaking changes for storybook's monorepo.